samples/python/gce/README.md

```diff
@@ -0,0 +1,95 @@
+# Snapshot Debugger Example for Python on Google Compute Engine
+
+NOTE: This sample application was copied from
+[getting-started-python/gce][sample-source]
+and modified for the Snapshot Debugger samples here.
+
+This folder contains the sample code for the [Deploying to Google Compute
```

Comment:
Would the name [Getting started with Python on Compute Engine] be better?

Reply:
Yes it would.  Updated.